accounts/migrations/0006_employer.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.4 on 2024-12-10 11:15
```

Comment:
before pushing your code, you should squash all migrations file in an app. don't add more than one file in a app to handle migrations.